packages/api-queries/src/site-activity-log-backup.ts

```diff
@@ -0,0 +1,13 @@
+import { fetchSiteBackupActivityLog } from '@automattic/api-core';
+import { queryOptions } from '@tanstack/react-query';
+
+export const siteBackupActivityLogEntriesQuery = (
+	siteId: number,
+	number: number = 1000, // 1000 is the maximum number of entries that can be fetched.
+	aggregate: boolean = false
+) =>
+	queryOptions( {
+		queryKey: [ 'site', siteId, 'backup-activity-log', number, aggregate ],
```

Comment:
Should we use `activity-log-backup` instead to keep it consistent with the filename?

```suggestion
		queryKey: [ 'site', siteId, 'activity-log-backup', number, aggregate ],
```

Reply:
Well, it is consistent with the query name `siteBackupActivityLogEntriesQuery`. I think we can keep it.